src/state/HeartwoodContextProvider.js

```diff
@@ -84,9 +87,11 @@ const HeartwoodContextProvider = ({ children }) => {
       // if the user is not currently logged in during a state change, check if they have a cookie
       if (!isLoggedIn(state.user)) {
         const endpoint = getEndpointPrefix() + "/verify"
-        fetch(endpoint, {method: "POST",
+        fetch(endpoint, {
+          method: "POST",
           headers: { "Content-Type": "application/json" },
-          body: JSON.stringify({ idt: getCookie("idt") })})
+          body: JSON.stringify({ idt: getCookie("idt") }),
+        })
```

Comment:
Codacy found an issue: [Unexpected line break before this closing brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5941503)